lib/ts/recipe/emailpassword/utils.ts

```diff
@@ -114,25 +119,58 @@ function validateAndNormaliseResetPasswordUsingTokenConfig(
     };
 }
 
-function normaliseSignInFormFields(formFields: NormalisedFormField[]) {
-    return formFields
-        .filter((filter) => filter.id === FORM_FIELD_EMAIL_ID || filter.id === FORM_FIELD_PASSWORD_ID)
-        .map((field) => {
-            return {
-                id: field.id,
-                // see issue: https://github.com/supertokens/supertokens-node/issues/36
-                validate: field.id === FORM_FIELD_EMAIL_ID ? field.validate : defaultValidator,
-                optional: false,
-            };
+function normaliseSignInFormFields(formFields?: TypeInputFormField[]) {
```

Comment:
If done this way, it'd be a breaking change: if someone added an override for their email validators in the sign up config, they'd expect it to be applied here. I'd prefer if we kept it that way as well.